v2/pkg/operators/common/dsl/dsl_test.go

```diff
@@ -309,12 +304,12 @@ func TestDateTimeDslExpressions(t *testing.T) {
 		now := time.Now()
 
 		dslExpressions := map[string]interface{}{
-			`date_time("%Y-%M-%D")`:                      fmt.Sprintf("%02d-%02d-%02d", now.Year(), now.Month(), now.Day()),
-			`date_time("%Y-%M-%D", unix_time())`:         fmt.Sprintf("%02d-%02d-%02d", now.Year(), now.Month(), now.Day()),
-			`date_time("%Y-%M-%D", 1642032000)`:          time.Date(2022, 01, 13, 0, 0, 0, 0, time.UTC).Local().Format("2006-01-02"),
-			`date_time("%H-%m")`:                         fmt.Sprintf("%02d-%02d", now.Hour(), now.Minute()),
-			`date_time("02-01-2006 15:04", unix_time())`: now.Format("02-01-2006 15:04"),
-			`date_time("02-01-2006 15:04", 1642032000)`:  time.Date(2022, 01, 13, 0, 0, 0, 0, time.UTC).Local().Format("02-01-2006 15:00"),
+			`date_time("%Y-%M-%D")`:                fmt.Sprintf("%02d-%02d-%02d", now.Year(), now.Month(), now.Day()),
```

Comment:
Were these tests failing? Because the time formatting was intentionally left there to cover all the possible scenarios.

Reply:
yeah, locally these were failing in some cases since we rely on minutes in the test hence moved to only hour